examples/pxScene2d/external/build.sh

```diff
@@ -54,23 +53,33 @@ fi
 
 #--------- OPENSSL
 
-cd ${OPENSSL_DIR}
-if [ "$(uname)" != "Darwin" ]
+if [ ! -e ${OPENSSL_DIR}/lib/libcrypto.dylib ] ||
+   [ "$(uname)" != "Darwin" ]
 then
-./config -shared  --prefix=`pwd`
-else
-./Configure darwin64-x86_64-cc -shared --prefix=`pwd`
+
+  cd ${OPENSSL_DIR}
+
+  if [ "$(uname)" != "Darwin" ]
+  then
+    ./config -shared  --prefix=`pwd`
+  else
+    ./Configure darwin64-x86_64-cc -shared --prefix=`pwd`
+  fi
+
+  make clean
+  make "-j${make_parallel}"
+  make install -i
+
+  rm -rf libcrypto.a
+  rm -rf libssl.a
+  rm -rf lib/libcrypto.a
+  rm -rf lib/libssl.a
+
+  cd ..
+
+  export LD_LIBRARY_PATH="${OPENSSL_DIR}/:$LD_LIBRARY_PATH"
+  export DYLD_LIBRARY_PATH="${OPENSSL_DIR}/:$DYLD_LIBRARY_PATH"
```

Comment:
These two variables should always be set (needed for other dependent external builds).  Please remove from the `if [ ! -e ${OPENSSL_DIR}/lib/libcrypto.dylib ]` check.